### ISSUE TITLE
Numpad DELETE not working [v1.12.0-beta0]

### ISSUE THREAD

**Issue by ozo2003** [atom/atom#12958]
### Description

Numpad DELETE not working

Updated to newest Atom Beta yesterday, DELETE key doesn't work in Atom anymore, all others on numpad do work (like HOME, END, PG DOWN). 
As far as I can see, newest update made changes to keymaps, but I think point was to improve, not remove features. 

**Expected behavior:** DELETE to work

**Actual behavior:** Nothing happens

### Versions
apm  1.12.5
npm  3.10.5
node 4.4.5
python 2.7.12
git 2.7.4

**Comment by Ben3eeE** [comment]
1. Can you reproduce this in safe mode `atom --safe`?
2. What operating system are you using?
3. What is your current keyboard layout?
4. What do you see in the keybinding resolver <kbd>Ctrl+.</kbd> when you press this button?

**Comment by ozo2003** [comment]
1. yes, I can
2. Ubuntu 16.04 64bit
3. qwerty en_US
4. "Key Binding Resolver:", nothing after :

**Comment by ozo2003** [comment]
It's the same behaviour for me with all layouts i have currently installed: en_US, lv_LV, ru_RU
second DELETE key works as expected, numpad DELETE is empty.
not using second cause it's in so unconvinient spot that i'm better of without it, numpad one is in great spot for me

**Comment by nathansobo** [comment]
I don't have a numeric keypad with a numlock key, so I'd appreciate it if someone could help me collect some data on this. Can someone open the Chromium dev tools in Atom `View > Developer > Toggle Developer Tools` and enter the following:

```js
document.addEventListener('keydown', e => console.log(e), true)
```

Then focus the window and press the delete key, then paste the information from the event? I'm particularly interested in the following fields: `key`, `code`, `keyCode`, `keyIdentifier`.

Thanks.

**Comment by N0r1x** [comment]
On the latest atom version from the .rpm package this problem does not occur, however on a built-from-source installation on the same computer it does occur.
![screenshot from 2016-10-18 00-37-42](https://cloud.githubusercontent.com/assets/20740526/19458279/697805c6-94cb-11e6-881e-9b2a076e35a7.png)

**Comment by nathansobo** [comment]
This will be fixed in 1.12.0-beta3, out later today.
